guides/common/modules/proc_configuring-how-many-processes-can-be-launched-by-apache-httpd.adoc

```diff
@@ -0,0 +1,25 @@
+[id="configuring_how_many_processes_can_be_launched_by_apache_httpd_{context}"]
+= Configuring how many processes can be launched by Apache httpd
+
+By default, HTTPD uses event request handling mechanism.
+When the number of requests to httpd exceeds the maximum number of child processes that can be launched to handle the incoming connections, httpd raises an HTTP 503 Service Unavailable error.
+Amidst httpd running out of processes to handle, the incoming connections can also result in multiple component failures on your {Project} services side due to the dependency of components like Pulp on the availability of httpd processes.
+
+You can adapt the configuration of HTTPD event to handle more concurrent requests based on your expected peak load.
+
+.Procedure
+. Modify the number of concurrent requests in '/etc/foreman-installer/custom-hiera.yaml' by changing or adding the following lines:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+apache::mod::event::serverlimit: 64
+apache::mod::event::maxrequestworkers: 1024
+apache::mod::event::maxrequestsperchild: 4000
+----
+. Apply your changes to {ProjectServer}.
+For more information, see xref:Applying_configurations_{context}[].
+
+The example is identical to running `{foreman-installer} --tuning=medium` on {ProjectServer}.
+
+.Note
+. But be warned that configuring these numbers in custom-hiera.yaml will effectively lock them, so if {Product} installer decides to change these numbers in the future by --tuning=* parameter, the possible change will be overwritten by this custom-hiera.yaml setting. Please use your numbers only if you have specific need for it.
```

Comment:
```suggestion
[NOTE]
====
But be warned that configuring these numbers in custom-hiera.yaml will effectively lock them, so if {Product} installer decides to change these numbers in the future by --tuning=* parameter, the possible change will be overwritten by this custom-hiera.yaml setting. Please use your numbers only if you have specific need for it.
====
```

two more things: please reword "be warned" and optionally choose a different admonition such as `CAUTION`.

Reply:
Perhaps a WARNING and cutting the initial sentence to "Configuring these numbers..."?

Reply:
Updated. Thanks :)